<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Item_edit</class>
 <widget class="QWidget" name="Item_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{background-color: rgb(255, 170, 0);}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);border: 3px solid #333</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.495 rgba(255, 255, 255, 255), stop:0.505 rgba(0, 255, 121, 255), stop:0.744318 rgba(0, 255, 196, 255), stop:1 rgba(0, 255, 229, 255));</string>
    </property>
    <property name="text">
     <string>物品图片</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>161</width>
      <height>181</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>物品名称</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>价格</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>属性1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>属性2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>属性3</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>输入信息</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.495 rgba(255, 255, 255, 255), stop:0.505 rgba(0, 255, 121, 255), stop:0.744318 rgba(0, 255, 196, 255), stop:1 rgba(0, 255, 229, 255));</string>
    </property>
    <property name="text">
     <string>物品属性</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>281</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.495 rgba(255, 255, 255, 255), stop:0.505 rgba(0, 255, 121, 255), stop:0.744318 rgba(0, 255, 196, 255), stop:1 rgba(0, 255, 229, 255));</string>
    </property>
    <property name="text">
     <string>可用素材</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.495 rgba(255, 255, 255, 255), stop:0.505 rgba(0, 255, 121, 255), stop:0.744318 rgba(0, 255, 196, 255), stop:1 rgba(0, 255, 229, 255));</string>
    </property>
    <property name="text">
     <string>选择素材路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Item_edit</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
